tests/containers/docker_image.pm

```diff
@@ -28,24 +28,37 @@ use containers::urls 'get_suse_container_urls';
 use version_utils qw(get_os_release check_os_release);
 
 sub run {
-    my ($image_names, $stable_names) = get_suse_container_urls();
     my ($running_version, $sp, $host_distri) = get_os_release;
     my $runtime = "docker";
 
     install_docker_when_needed($host_distri);
     allow_selected_insecure_registries(runtime => $runtime);
     scc_apply_docker_image_credentials() if (get_var('SCC_DOCKER_IMAGE'));
 
-    for my $iname (@{$image_names}) {
-        test_container_image(image => $iname, runtime => $runtime);
-        test_rpm_db_backend(image => $iname, runtime => $runtime);
-        build_container_image(image => $iname, runtime => $runtime);
-        if (check_os_release('suse', 'PRETTY_NAME')) {
-            test_opensuse_based_image(image => $iname, runtime => $runtime);
-            build_with_zypper_docker(image => $iname, runtime => $runtime);
-        }
-        else {
-            exec_on_container($iname, $runtime, 'cat /etc/os-release');
+    # We may test either one specific VERSION
+    # or comma-separated CONTAINER_VERSIONS
+    my $versions;
+    if (get_var('CONTAINER_VERSIONS') =~ /,/) {
+        $versions = get_var('CONTAINER_VERSIONS');
+    } else {
+        $versions = get_required_var('VERSION');
+    }
+
+    for my $version (split(/,/, $versions)) {
+        record_info "VERSION", "We are testing $version now.";
```

Comment:
```suggestion
        record_info "IMAGE", "Starting tests for the image $version";
```

Reply:
It looks like this suggestion was lost in one of the recent updates